src/ui/components/Rating/index.js

```diff
@@ -62,15 +62,14 @@ export class RatingBase extends React.Component {
           'Rating-half-star': (rating && thisRating > rating &&
             thisRating - 0.5 <= rating),
         }),
-        id: `Rating-rating-${thisRating}`,
```

Comment:
I guess I said conditionally add, but it'd be nice to keep all the props in one place. Could this be:

```js
id: !readOnly ? `Rating-rating-${thisRating}` : null,
```
If so I think that'd be the easiest to follow.